src/component/Nav/Nav.js

```diff
@@ -1,11 +1,68 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Categories from './Categories/Categories';
+import './Nav.scss';
 
 export class Nav extends Component {
   render() {
     return (
-      <div>
-        <h1>nav</h1>
-      </div>
+      <header className="header">
+        <nav className="nav">
+          <div className="logoWrap">
+            <img src="./images/home.png" alt="logoImg" className="logoImg" />
+            <span className="logo">꾸며줘홈즈</span>
+          </div>
+          <div className="menuLeft">
+            <div className="categoryMenu">
+              <div className="categoryMenuHover">
+                <div className="categoryMenuImg" />
+                <span className="categoryMenuName">카테고리</span>
+              </div>
+              {/* 상수 데이터와 함께 컴포넌트로 따로 뺄 수 있도록 */}
```

Comment:
```suggestion
              // TODO: 상수 데이터와 함께 컴포넌트로 따로 빼기
```
나중에 고쳐야 하는 경우나 더 해야 하는 경우는 주석에 TODO, TOFIX 등의 preset 을 붙이는 것이 알아보고 검색해서 찾아오기 더 쉽습니다!

Reply:
적으신대로 반복되는 UI 는 map을 이용해서 표현할 수 있도록 해보세요!